presentation/src/main/java/com/globant/equattrocchio/cleanarchitecture/mvp/presenter/ImagesPresenter.java

```diff
@@ -5,83 +5,82 @@
 import com.globant.equattrocchio.cleanarchitecture.util.bus.RxBus;
 import com.globant.equattrocchio.cleanarchitecture.mvp.view.ImagesView;
 import com.globant.equattrocchio.cleanarchitecture.util.bus.observers.CallServiceButtonObserver;
-import com.globant.equattrocchio.data.ImagesServicesImpl;
-import com.globant.equattrocchio.domain.GetJsonUseCase;
+import com.globant.equattrocchio.data.response.Result;
 import com.globant.equattrocchio.domain.GetLatestImagesUseCase;
+import com.globant.equattrocchio.domain.enities.ImageEntity;
+
+import java.util.List;
 
 import io.reactivex.annotations.NonNull;
-import io.reactivex.observers.DefaultObserver;
 import io.reactivex.observers.DisposableObserver;
 
 public class ImagesPresenter {
 
     private ImagesView view;
-//    private GetLatestImagesUseCase getLatestImagesUseCase;
-    private GetJsonUseCase getJsonUseCase;
 
-//    public ImagesPresenter(ImagesView view, GetLatestImagesUseCase getLatestImagesUseCase) {
+//    This is from the second Module's item
+//    private GetJsonUseCase getJsonUseCase;
```

Comment:
remove it, it's fine not having code for second module, there's already a PR for that

Reply:
ok done